install.sh

```diff
@@ -47,12 +47,6 @@ git submodule init
 git submodule sync
 git submodule update --init --recursive
 
-# Get react native
-pushd "libs/SalesforceReact"
-rm -rf node_modules
-npm install
-popd
-
 
 # Remove the old Xcode templates, if they still exist.
 hybrid_template_dir="${HOME}/Library/Developer/Xcode/Templates/Project Templates/Application/Hybrid Force.com App.xctemplate"
```

Comment:
This is the only remaining mention of hybrid, but it didn't seem like it should be removed.

Reply:
It can be removed. It's almost 5 yrs old.